docs/source/install/ansible.rst

```diff
@@ -220,7 +220,7 @@ functionality to specific users:
                     - system_admin
 
         - name: Verify EWC Installation
-          role: bwc_smoketests
+          role: ewc_smoketests
```

Comment:
Thanks for finding these!

I thought I updated st2docs per https://github.com/StackStorm/ansible-st2/pull/223#issuecomment-449152890